.github/workflows/test_on_push.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
         # it uses its own metrics package.
         # We test old metrics with Cartridge 2.7.9 because since 2.8.0 it
         # requires metrics 1.0.0.
-        tarantool-version: ["1.10.6", "1.10", "2.2", "2.3", "2.4", "2.5", "2.6", "2.7", "2.8", "2.10", "2.11"]
+        tarantool-version: ["1.10.6", "1.10", "2.8", "2.10", "2.11"]
```

Comment:
What do you mean under 'some'?
tarantool 2.8 has reached it EOL. 
tarantool 1.10.x has reached it's EOS and EOL.
Do we keep them because many users use crud on them?

Reply:
Yes. It's more like "some users run crud on 1.10.15 and 2.8" rather than "many". On the other hand, as far as I know, there should be almost none 2.2-2.7 users since they had migrated to 2.8-2.11.